services/brig/src/Brig/User/Search/Index/Types.hs

```diff
@@ -60,7 +61,9 @@ data IndexUser = IndexUser
     _iuManagedBy :: Maybe ManagedBy,
     _iuCreatedAt :: Maybe UTCTime,
     _iuRole :: Maybe Role,
-    _iuSearchVisibilityInbound :: Maybe SearchVisibilityInbound
+    _iuSearchVisibilityInbound :: Maybe SearchVisibilityInbound,
+    _iuScimExternalId :: Maybe Text,
+    _iuSso :: Maybe Sso
```

Comment:
This may be correct (I don't know ES), but are you sure if a field is not indexed, it should still appear here and not just in `UserDoc`?  If so, why?

Reply:
AFAIK `UserDoc` is a subset of the index.